Libraries/CMSIS/Device/Maxim/MAX32650/Include/aes_key_regs.h

```diff
@@ -78,21 +77,21 @@ extern "C" {
  * @ingroup     aes_key
  * @defgroup    aes_key_registers AES_KEY_Registers
  * @brief       Registers, Bit Masks and Bit Positions for the AES_KEY Peripheral Module.
- * @details     AES Key Registers.
+ * @details AES Key Registers.
  */
 
 /**
  * @ingroup aes_key_registers
  * Structure type to access the AES_KEY Registers.
  */
 typedef struct {
-    __IO uint32_t aes_key0;             /**< <tt>\b 0x000:</tt> AES_KEY AES_KEY0 Register */
-    __R  uint32_t rsv_0x4_0x7f[31];
-    __IO uint32_t aes_key1;             /**< <tt>\b 0x080:</tt> AES_KEY AES_KEY1 Register */
-    __R  uint32_t rsv_0x84_0xff[31];
-    __IO uint32_t aes_key2;             /**< <tt>\b 0x100:</tt> AES_KEY AES_KEY2 Register */
-    __R  uint32_t rsv_0x104_0x17f[31];
-    __IO uint32_t aes_key3;             /**< <tt>\b 0x180:</tt> AES_KEY AES_KEY3 Register */
+    __IO uint32_t aes_key0[4];          /**< <tt>\b 0x00:</tt> AES_KEY AES_KEY0 Register */
+    __R  uint32_t rsv_0x10_0x1f[4];
+    __IO uint32_t aes_key1[4];          /**< <tt>\b 0x20:</tt> AES_KEY AES_KEY1 Register */
+    __R  uint32_t rsv_0x30_0xff[52];
+    __IO uint32_t aes_key2[4];          /**< <tt>\b 0x100:</tt> AES_KEY AES_KEY2 Register */
+    __R  uint32_t rsv_0x110_0x17f[28];
+    __IO uint32_t aes_key3[4];          /**< <tt>\b 0x180:</tt> AES_KEY AES_KEY3 Register */
```

Comment:
This is register file is outdated. Use the register SVD scripts to regenerate this file so you don't have to make all these changes.